docs/organizations/verify-user-permissions.mdx

```diff
@@ -74,139 +76,177 @@ The examples below work for both SSR and CSR. Examples are written for Next.js A
 
 ## Authorization in React Server Components
 
-<Tabs type="framework" items={[ "RSC with has()", "RSC with auth().protect()", "RSC with <Protect>"]}>
+<Tabs items={["<Protect>", "has()", "auth().protect()"]}>
   <Tab>
-    The following example uses `has()` to inspect a user's permissions granularly. If the user doesn't have the `org:team_settings:read` permission, the component returns `null` instead of rendering its children.
+      The following example uses Clerk's `<Protect>` component to only render the form for users with the `org:team_settings:read` permission. If the user is not authorized, the component will not render its children.
+
+      ```tsx {{ filename: '/app/dashboard/settings/layout.tsx' }}
+        import type { PropsWithChildren } from "react";
+        import { Protect } from "@clerk/nextjs";
 
-    ```tsx filename="/app/dashboard/settings/layout.tsx"
+        export default function SettingsLayout(props: PropsWithChildren){
+          return (
+            <Protect
+              permission="org:team_settings:read"
+            >
+              {props.children}
+            </Protect>
+          )
+        }
+      ```
+  </Tab>
+
+  <Tab>
+    The following example uses `has()` to inspect a user's permissions granularly. If the user doesn't have the `org:team_settings:read` permission, `has()` returns `false`, causing the component to return `null` instead of rendering its children.
+
+    ```tsx {{ filename: '/app/dashboard/settings/layout.tsx' }}
       import type { PropsWithChildren } from "react";
       import { auth } from "@clerk/nextjs/server";
 
       export default function SettingsLayout(props: PropsWithChildren){
-        const { has } = auth()
+        const { has } = auth();
 
-        const canAccessSettings = has({permission: "org:team_settings:read"});
+        const canAccessSettings = has({ permission: "org:team_settings:read" });
 
         if(!canAccessSettings) return null;
 
-        return props.children
+        return props.children;
       }
       ```
 
   </Tab>
 
   <Tab>
-    <Callout>
-      `auth().protect()` will throw a `notFound` Next.js error if authorization checks fail.
-    </Callout>
+      <Callout type="warning">
```

Comment:
Does it make a difference if we use `<Callout>` rather than our new callout syntax?

Reply:
probably should use the new syntax ! i'll remove the tech debt